### ISSUE TITLE
Platt Scaling Code

### ISSUE THREAD

**Issue by gvanhorn38** [visipedia/tf_classification#3]
Write a function that takes as input training logits & labels and validation logits & labels and fits a logistic regression model to the scores. 

It would be convenient if this function could modify a network to do the learned transformation.

Write a tutorial explaining why you would want to do this.

**Comment by gvanhorn38** [comment]
Tried some experiments to learn a parameter (scalar and a vector) to rescale the logits using an extra validation dataset. Doesn't seem to be worth while. For now lets just keep it simple and use the outputs of the softmax.
